src/py42/_internal/clients/storage/storagenode.py

```diff
@@ -0,0 +1,29 @@
+from py42.clients import BaseClient
+import requests
+
+
+class StorageNodeClient(BaseClient):
+
+    _base_uri = u"c42api/v3/"
+
+    def __init__(self, session, archive_guid, file_id, timestamp):
+        super(StorageNodeClient, self).__init__(session)
+        self._archive_guid = archive_guid
+        self._file_id = file_id
+        self._timestamp = timestamp
+
+    def get_download_token(self):
+
+        params = "archiveGuid={0}&fileId={1}&versionTimestamp={2}".format(
```

Comment:
in order to ensure url escaping works properly, make `params` a dict, and pass it to the `params` arg of `get`:
```python
params = {
    u"archiveGuid": self._archive_guid,
    u"fileId": self._file_id,
    u"versionTimestamp": self._timestamp
}
uri = "{0}{1}".format(self._base_uri, u"FileDownloadToken")
return self._session.get(uri, params=params)
    
``

Reply:
Done.